recipes/icu/all/conanfile.py

```diff
@@ -61,15 +66,45 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
+    def validate(self):
+        if not bool(self._platform):
+            raise ConanInvalidConfiguration(
+                "Compiling ICU for {} with {} not supported yet".format(str(self.settings.os),
+                                                                        str(self.settings.compiler)))
+
+    @property
+    def _platform(self):
+        return {
+            ("Windows", "Visual Studio"): "Cygwin/MSVC",
+            ("Windows", "gcc"): "MinGW",
+            ("AIX", "gcc"): "AIX/GCC",
```

Comment:
I don't know whether this is in-scope, but this is missing a line for Android.
https://github.com/unicode-org/icu/blob/b03b8be7414422d8198b77da4c7ed8c9457dd349/icu4c/source/runConfigureICU#L48
```suggestion
            ("Android", "clang"): "Linux",
            ("AIX", "gcc"): "AIX/GCC",
```

Reply:
Yes it's not in scope of this PR, and I've not tested Android.

Reply:
I just tried it with the icu recipe on current master, and it's working.
```patch
--- a/recipes/icu/all/conanfile.py
+++ b/recipes/icu/all/conanfile.py
@@ -136,6 +136,7 @@ class ICUBase(ConanFile):
         prefix = self.package_folder.replace("\\", "/")
         platform = {("Windows", "Visual Studio"): "Cygwin/MSVC",
                     ("Windows", "gcc"): "MinGW",
+                    ("Android", "clang"): "Linux",
                     ("AIX", "gcc"): "AIX/GCC",
                     ("AIX", "xlc"): "AIX",
                     ("SunOS", "gcc"): "Solaris/GCC",
```

When I run `conan create . icu/69.1@ -pr:b default64 -pr:h android22b`, icu is build twice: once for x86_64 and once for android.
Do you know how I can avoid rebuilding the x86_64 package when running this command?
For speed purposes.

Reply:
`-b missing` will skip building everything and just run the test package.

Reply:
Unfortunately, there are no `-b:h` and `-b:b`.

Reply:
Would be nice to have that as a time saver.

Reply:
You may also have to override host triplet if Android like I did if iOS/tvOS/watchOS.